### PR TITLE
o-rly-typst:0.1.0

### PR DESCRIPTION
**Pull request by dei-layborer**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A package for creating O'Rly-/O'Reilly-like cover pages.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

NOTE: I goofed and accidentally changed the `LICENSE` for the *repository* rather than just my submission to it.  I changed it back, so hopefully that doesn't cause any issues.

### CODE REVIEW

File: LICENSE
Comment (elegaanz) [inline LICENSE:1]: Can you please revert the changes made to this file?

**Review by elegaanz — COMMENTED** [review]
Hello and thank you for the package submission :) Can you please change the name of the package, to something less canonical for what the package does, and that does not include the word "typst"?

File: packages/preview/o-rly-typst/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name = "o-rly-typst"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Dei Layborer"]
+license = "GPL-3.0"
+compiler = "0.11.1"
+description = "A package for creating O'Rly- / O'Reilly-type cover pages"
+repository = "https://github.com/dei-layborer/o-rly-typst"
+categories = ["components", "layout", "fun"]
+exclude = ["$$$-gimmie.svg", "made-with-(2s)-2,6-diamino-n-[(2s)-1-phenylpropan-2-yl]hexanamide-n-[(2s)-1-phenyl-2-propanyl]-l-lysinamide.svg", "example.png"]
```
Comment (elegaanz) [inline packages/preview/o-rly-typst/0.1.0/typst.toml:11]: I think the second file name here have changed in the repository.

File: LICENSE
Comment (dei-layborer) [reply LICENSE:1]: Yes, I was trying to find the "revert" option, but couldn't :)  In light of this and the other issues, I'm going to re-do the pull request.

File: packages/preview/o-rly-typst/0.1.0/typst.toml (hunk 1 above)
Comment (dei-layborer) [reply packages/preview/o-rly-typst/0.1.0/typst.toml:11]: Thanks for catching this, you're right ... the build process seemed to take issue with the filename.

**Comment by dei-layborer** [comment]
I'm going to re-do the pull request once I've cleaned everything up.  Thanks!
